RPIPs/RPIP-49.md

```diff
@@ -120,14 +121,21 @@ The tokenomics rework package will likely be split between two protocol upgrades
   * Heuristic adjustments.
 
 ## Current Status
-Last Updated: July 3rd
+Last Updated: July 16th
 
 Current efforts are primarily focused on:
-1. Updating and polishing the RPIP specifications in response to feedback.
-2. Attempting to reduce the scope of the required implementation work to improve time-to-market without overly impacting contents.
+1. Confirming Core Team is happy with RPIP specification contents and making minor adjustments as needed.
+2. Polishing the RPIP specifications ready for vote.
 
 A maintained list of open tasks is available via google sheet [here](https://docs.google.com/spreadsheets/d/1KDTeFnNl3XLFO37upti6NbT2p2GDYJ4GKH4aJ51gQZA/edit?pli=1#gid=725857744). 
 
+### Still To Ratfiy Prior to Saturn 1
+These items are to be considered flexible until ratified explicitly, rather than ratified alongside the tokenomics rework package as a whole. 
+
+* **Saturn 2 Surplus Share Strategy** - Decide and ratify how to manage surplus share in Saturn 2 (we want to decide this before Saturn 1 launches to avoid the incumbent advantage for the voter_share option.)
+* **Deposit Strategy** - Decide whether to take a 2TX or 3TX deposit strategy for validator creation and ratify choice.
+* **Maximum Megapool ETH Penalty** - Decide and ratify a non-placeholder value for the maximum penalty, likely needs community research to find a good value.
```

Comment:
I think knoshua has convinced me this is too narrow for our current confidence level. Maybe more like 
> **Penalty System** - Research, draft and ratify a penalty system. Right now, there is an idea written, with a missing Maximum ETH Penalty. There has been some discussion around, eg, making this a 2 step "freeze and then determine penalty" process. Work is needed, but this doesn't change the overall proposal much at all.

Reply:
Have added it in an abbreviated form, I don't think we need to go into significant detail here.